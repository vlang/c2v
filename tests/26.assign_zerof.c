int main() {
    double a = 0.0f;
}