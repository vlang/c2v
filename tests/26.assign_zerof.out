@[translated]
module main

fn main() {
	a := f64(0)
}
